lib/elixir/lib/kernel.ex

```diff
@@ -566,6 +566,9 @@ defmodule Kernel do
       iex> is_atom("true")
       false
 
+      iex> is_atom(AnAtom)
+      true
+
   """
```

Comment:
```suggestion
      iex> is_atom(AnAtom)
      true
      
      iex> is_atom(AnAtom)
      true
  """
```

Keeping true and false results separate